docs/docs-source/docs/modules/ROOT/pages/index.adoc

```diff
@@ -62,8 +62,9 @@ Cloudflow includes tools for developing and deploying streaming applications:
 
 * For deploying to Kubernetes clusters, the Cloudflow operator takes care of orchestrating the deployment of the different parts of a streaming pipeline as an end-to-end application.
 
-Cloudflow dramatically accelerates your application development efforts, reducing the time required to create, package, and deploy an application from weeks--to hours. 
-Lightbend also offers support with a https://www.lightbend.com/lightbend-platform-subscription[Lightbend Subscription].
+Lightbend's commercial version of Cloudflow, https://developer.lightbend.com/docs/cloudflow/current/index.html[Akka Data Pipelines], includes support along with https://developer.lightbend.com/docs/telemetry/current/home.html[Telemetry] and https://developer.lightbend.com/docs/cloudflow/current/console/index.html[Lightbend Console], which provide visibility into your running Cloudflow apps.
+
+You can also use https://developer.lightbend.com/docs/cloudflow/current/gcp/index.html[Akka Data Pipelines on Google Cloud Platform]. 
```

Comment:
I would remove this for now it is not ready.

Reply:
The version on GCP Marketplace is outdated and can only be installed on an existing cluster.

Reply:
OK, I'll comment out for now.